[2024-08-24T12:32:17.392+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-24T12:32:17.421+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bybit_tick_fetcher.fetch_and_store_task manual__2024-08-24T12:32:16.684800+00:00 [queued]>
[2024-08-24T12:32:17.434+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bybit_tick_fetcher.fetch_and_store_task manual__2024-08-24T12:32:16.684800+00:00 [queued]>
[2024-08-24T12:32:17.435+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-24T12:32:17.469+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): fetch_and_store_task> on 2024-08-24 12:32:16.684800+00:00
[2024-08-24T12:32:17.493+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1943) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-24T12:32:17.494+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bybit_tick_fetcher', 'fetch_and_store_task', 'manual__2024-08-24T12:32:16.684800+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmptp_ay8r4']
[2024-08-24T12:32:17.497+0000] {standard_task_runner.py:72} INFO - Started process 1944 to run task
[2024-08-24T12:32:17.498+0000] {standard_task_runner.py:105} INFO - Job 53: Subtask fetch_and_store_task
[2024-08-24T12:32:17.575+0000] {task_command.py:467} INFO - Running <TaskInstance: bybit_tick_fetcher.fetch_and_store_task manual__2024-08-24T12:32:16.684800+00:00 [running]> on host bf8e87922500
[2024-08-24T12:32:17.756+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bybit_tick_fetcher' AIRFLOW_CTX_TASK_ID='fetch_and_store_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-24T12:32:16.684800+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-24T12:32:16.684800+00:00'
[2024-08-24T12:32:17.759+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-08-24T12:32:17.760+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-08-24T12:32:17.761+0000] {logging_mixin.py:190} INFO - Current task name:fetch_and_store_task state:running start_date:2024-08-24 12:32:17.422390+00:00
[2024-08-24T12:32:17.762+0000] {logging_mixin.py:190} INFO - Dag name:bybit_tick_fetcher and current dag run status:running
[2024-08-24T12:32:17.763+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-24T12:32:17.765+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-23.csv.gz
[2024-08-24T12:34:30.564+0000] {data_loader.py:54} ERROR - Error writing file /Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-23.csv.gz: [Errno 2] No such file or directory: '/Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-23.csv.gz'
[2024-08-24T12:34:30.571+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-24.csv.gz
[2024-08-24T12:35:36.544+0000] {data_loader.py:54} ERROR - Error writing file /Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-24.csv.gz: [Errno 2] No such file or directory: '/Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-24.csv.gz'
[2024-08-24T12:35:36.547+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-25.csv.gz
[2024-08-24T12:36:31.277+0000] {data_loader.py:54} ERROR - Error writing file /Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-25.csv.gz: [Errno 2] No such file or directory: '/Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-25.csv.gz'
[2024-08-24T12:36:31.279+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-26.csv.gz
[2024-08-24T12:36:47.969+0000] {data_loader.py:54} ERROR - Error writing file /Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-26.csv.gz: [Errno 2] No such file or directory: '/Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-26.csv.gz'
[2024-08-24T12:36:47.972+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-27.csv.gz
[2024-08-24T12:37:17.525+0000] {data_loader.py:54} ERROR - Error writing file /Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-27.csv.gz: [Errno 2] No such file or directory: '/Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-27.csv.gz'
[2024-08-24T12:37:17.537+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-28.csv.gz
[2024-08-24T12:59:34.209+0000] {job.py:229} INFO - Heartbeat recovered after 1330.86 seconds
[2024-08-24T12:59:47.870+0000] {data_loader.py:39} ERROR - Error fetching https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-28.csv.gz: ('Connection broken: IncompleteRead(2948633 bytes read, 13210883 more expected)', IncompleteRead(2948633 bytes read, 13210883 more expected))
[2024-08-24T12:59:47.873+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-29.csv.gz
[2024-08-24T13:01:06.282+0000] {data_loader.py:54} ERROR - Error writing file /Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-29.csv.gz: [Errno 2] No such file or directory: '/Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-29.csv.gz'
[2024-08-24T13:01:06.286+0000] {data_loader.py:34} INFO - Download start for https://public.bybit.com/trading/BTCUSDT/BTCUSDT2023-08-30.csv.gz
[2024-08-24T13:01:42.209+0000] {data_loader.py:54} ERROR - Error writing file /Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-30.csv.gz: [Errno 2] No such file or directory: '/Users/farhan/Desktop/Data/BTCUSDT/BTCUSDT2023-08-30.csv.gz'
[2024-08-24T13:01:42.212+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-24T13:01:42.226+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-24T13:01:42.227+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bybit_tick_fetcher, task_id=fetch_and_store_task, run_id=manual__2024-08-24T12:32:16.684800+00:00, execution_date=20240824T123216, start_date=20240824T123217, end_date=20240824T130142
[2024-08-24T13:01:42.236+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-08-24T13:01:42.236+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-08-24T13:01:42.237+0000] {logging_mixin.py:190} INFO - Dag name:bybit_tick_fetcher queued_at:2024-08-24 12:32:16.704003+00:00
[2024-08-24T13:01:42.237+0000] {logging_mixin.py:190} INFO - Task hostname:bf8e87922500 operator:PythonOperator
[2024-08-24T13:01:42.286+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-24T13:01:42.299+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-24T13:01:42.300+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
